torch/ao/quantization/fx/convert.py

```diff
@@ -456,6 +458,23 @@ def convert_weighted_module(
         parent_name, name = _parent_name(node.target)
         setattr(modules[parent_name], name, ref_qmodule)
 
+def _remove_previous_dequantize_in_custom_module(node: Node, prev_node: Node, graph: Graph):
+    """
+    Change the connection for custom module, we'll change the input
+    of custom module node to quantize node:
+    Before: quantize - dequantize - custom - module
```

Comment:
nit: could we clarify what this graph is doing? Is `custom - module` a single node or two nodes?

Reply:
Sure. I just moved this comment but I can clarify it. I think `custom - module` is supposed to be a single node